tracim/lib/core/group.py

```diff
@@ -14,9 +16,12 @@ def __init__(
             self,
             session: Session,
             current_user: typing.Optional[User],
+            config: CFG
+
     ):
         self._user = current_user
         self._session = session
+        self._config = config
```

Comment:
The goal is to have same params in entry for all internal API, because we may need _config later. If we don't do it now, we should add it later and change many tests...